### PR TITLE
Refactor profile signaling provisions

### LINKED ISSUE #331: Document structure of guidance for other profiles
In sections 7.9.3 and 7.9.4, we have non-normative profile setting recommendations for other profiles (EBU-TT-D and SDP-US).  And, we have Annex I which contains related non-normative language for compatibility with various external profiles, including the two in 7.9. 

Conversely, we have normative profile settings for IMSC1 in 7.9.2 and related non-normative information in I.6.

All of this text would be a lot more readable if it were together. For example, Annex I talks about the relationship of the profiles to IMSC1.1, which is key to understanding why one might set the profile settings the way they are.

To improve readability, I propose the following restructuring of the sections: 1. move 7.9.3 and 7.9.4 to the ends of I.2 and I.3 respectively; and 2. move I.6 to the beginning of 7.9.2.

This also aggregates normative and non-normative text in the body (IMSC1) and the Annex (everything else) respectively.

### PR DESCRIPTION
**Pull request by palemieux**
Close #331

Built at https://rawgit.com/w3c/imsc/issue-331-refactor-profile-signaling/imsc1/spec/ttml-ww-profiles.html

Diff at https://services.w3.org/htmldiff?doc1=https%3A%2F%2Frawgit.com%2Fw3c%2Fimsc%2FIMSC1.1%2Fimsc1%2Fspec%2Fttml-ww-profiles.html&doc2=https%3A%2F%2Frawgit.com%2Fw3c%2Fimsc%2Fissue-331-refactor-profile-signaling%2Fimsc1%2Fspec%2Fttml-ww-profiles.html

### CODE REVIEW

**Comment by tairt** [comment]
You may also adjust the sample in section `I.2 EBU-TT-` and omit  `ebuttm:documentMetadata` as follows

NOW

```xml
        <metadata>
            <ebuttm:documentMetadata>
                <ebuttm:conformsToStandard>urn:ebu:tt:distribution:2014-01</ebuttm:conformsToStandard>
                <ebuttm:conformsToStandard>http://www.w3.org/ns/ttml/profile/imsc1/text</ebuttm:conformsToStandard>
                <ebuttm:conformsToStandard>http://www.w3.org/ns/ttml/profile/imsc1.1/text</ebuttm:conformsToStandard>
            </ebuttm:documentMetadata>
        </metadata>
```

AFTER

```xml
        <metadata>
                <ebuttm:conformsToStandard>urn:ebu:tt:distribution:2014-01</ebuttm:conformsToStandard>
                <ebuttm:conformsToStandard>http://www.w3.org/ns/ttml/profile/imsc1/text</ebuttm:conformsToStandard>
                <ebuttm:conformsToStandard>http://www.w3.org/ns/ttml/profile/imsc1.1/text</ebuttm:conformsToStandard>
        </metadata>
```

File: imsc1/spec/ttml-ww-profiles.html [hunk 1]
```diff
@@ -2635,109 +2636,35 @@ <h4>itts:fillLineGap</h4>
...
-          <li>the <code>ttp:contentProfiles</code> attribute SHALL NOT simultaneously signal conformance to both <a>Image
-          Profile</a> and <a>Text Profile</a>;
-          </li>
-
-          <li>the <code>ttp:profile</code> attribute SHALL NOT be present; and</li>
-
-          <li>the <code>ttp:profile</code> element SHALL NOT be present.</li>
-        </ul>
+        <p class="note">To minimize the risk of incompatibility, the use of the <code>ttp:contentProfiles</code> attribute and
+        other [[TTML2]] syntax is not recommended in documents intended to conform to profiles that are based on [[TTML1]], e.g.
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html]: The `[[TTML2]] syntax` is ambiguous since TTML2 syntax includes TTML1 syntax. Suggest rewording to `other syntax introduced in [[TTML2]]`

File: imsc1/spec/ttml-ww-profiles.html [hunk 2]
```diff
@@ -2635,109 +2636,35 @@ <h4>itts:fillLineGap</h4>
...
-        <p class="note">The scenarios specified in Sections <a href="#general-signaling"></a>, <a href="#imsc1-compat"></a>,
-        <a href="#ebu-tt-d-compat"></a> and <a href="#sdp-us-compat"></a> are intended to be mutually exclusive.</p>
+        <p class="note"><a href="#profile-resolution"></a> specifies the process by which the effective processor profile
+        associated with the <a>Document Instance</a> is determined.</p>
       </section>
 
-      <section id="imsc1-compat">
-        <h4>IMSC1 Compatibility</h4>
+      <section id="ebu-tt-signaling">
+        <h4>Signaling in EBU-TT documents</h4>
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html]: The EBU-TT metadata can be used in any document, including EBU-TT and EBU-TT-D, so the title here seems wrong. (I think "signalling" is misspelled too, though it could be a British/US English difference). Suggest "`Signalling conformance using EBU-TT metadata`" as an alternative.

File: imsc1/spec/ttml-ww-profiles.html [hunk 3]
```diff
@@ -4335,18 +4262,21 @@ <h2>Compatibility with other TTML-based specifications (non-normative)</h2>
...
       <h3>Overview</h3>
 
-      <p>This specification is designed to be compatible with [[!SMPTE2052-1]], [[!EBU-TT-D]] and [[ttml10-sdp-us]]. Specifically,
-      it is possible to create a document that:</p>
+      <p>This specification is designed to be compatible with [[ttml-imsc1]], [[ttml-imsc1.0.1]], [[SMPTE2052-1]], [[EBU-TT-D]] and
+      [[ttml10-sdp-us]]. Specifically, by selecting a subset of the features and extensions defined in this specification, it is
+      possible to create a document that:</p>
 
       <ul>
         <li>conforms to one of [[ttml-imsc1]], [[ttml-imsc1.0.1]],[[!SMPTE2052-1]], [[!EBU-TT-D]] or [[ttml10-sdp-us]], and also
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html]: s/conforms to one of/conforms to one or more of

File: imsc1/spec/ttml-ww-profiles.html [hunk 4]
```diff
@@ -4458,7 +4385,7 @@ <h3>CFF-TT</h3>
         </li>
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html]: On line 4451 there's a typo that may as well get fixed here - s/does not specif**ies**/does not specif**y**

File: imsc1/spec/ttml-ww-profiles.html [hunk 5]
```diff
@@ -4458,7 +4385,7 @@ <h3>CFF-TT</h3>
         </li>
 
         <li>[[CFF]] requires the use of the <code>ttp:profile</code> element, whereas this specification recommends the use of the
-        <code>ttp:profile</code> attribute.</li>
+        <code>ttp:contentProfiles</code> attribute.</li>
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html]: But this specification does still permit `ttp:profile`, so that does not _require_ processing, right?

File: imsc1/spec/ttml-ww-profiles.html [hunk 6]
```diff
@@ -4470,6 +4397,17 @@ <h3>IMSC1 and IMSC1.0.1</h3>
       <p>The Image Profile is a strict superset of the [[ttml-imsc1]] and [[ttml-imsc1.0.1]] Image Profile.</p>
 
       <p>A document that conforms to [[ttml-imsc1]] or [[ttml-imsc1.0.1]] therefore also conforms to this specification.</p>
+
+      <p>A number of features that previously permitted in [[ttml-imsc1.0.1]] have been designated as permitted-deprecated in the
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html]: s/that previously permitted/that **were** previously permitted

Comment (nigelmegitt) [reply imsc1/spec/ttml-ww-profiles.html]: s/**have been** designated/**are** designated

File: imsc1/spec/ttml-ww-profiles.html [hunk 7]
```diff
@@ -4470,6 +4397,17 @@ <h3>IMSC1 and IMSC1.0.1</h3>
       <p>The Image Profile is a strict superset of the [[ttml-imsc1]] and [[ttml-imsc1.0.1]] Image Profile.</p>
 
       <p>A document that conforms to [[ttml-imsc1]] or [[ttml-imsc1.0.1]] therefore also conforms to this specification.</p>
+
+      <p>A number of features that previously permitted in [[ttml-imsc1.0.1]] have been designated as permitted-deprecated in the
+      <a>Text Profile</a> and <a>Image Profile</a>. [[ttml-imsc1.0.1]] documents that use such features are therefore susceptible
+      to be incompatible with future versions of this specification.</p>
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html]: Grammar: s/to **be**/to **being**

File: imsc1/spec/ttml-ww-profiles.html [hunk 8]
```diff
@@ -4470,6 +4397,17 @@ <h3>IMSC1 and IMSC1.0.1</h3>
...
 
       <p>A document that conforms to [[ttml-imsc1]] or [[ttml-imsc1.0.1]] therefore also conforms to this specification.</p>
+
+      <p>A number of features that previously permitted in [[ttml-imsc1.0.1]] have been designated as permitted-deprecated in the
+      <a>Text Profile</a> and <a>Image Profile</a>. [[ttml-imsc1.0.1]] documents that use such features are therefore susceptible
+      to be incompatible with future versions of this specification.</p>
+
+      <p>The following is an example of a document that conforms to both <a>Image Profile</a> and [[ttml-imsc1.0.1]] Image
+      Profile:</p>
+      <pre class='example' data-include='examples/imsc1-image-example.xml' data-include-format='text'>
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html]: The example includes syntax that is now _permitted-deprecated_ and therefore may be incompatible with future versions as indicated in the previous paragraph. I think we need to add text to make it super clear that this example includes those.

**Comment by palemieux** [comment]
@nigelmegitt Are you happy with the edits?

File: imsc1/spec/ttml-ww-profiles.html (hunk 4 above)
Comment (nigelmegitt) [reply imsc1/spec/ttml-ww-profiles.html]: Thanks, this is resolved but can't show as outdated in the GitHub UI because it wasn't part of the original change.
